packages/cli/src/test-utils/cliUtils.ts

```diff
@@ -15,5 +15,9 @@ export async function testLocally(
   return command.run(extendedArgv, config)
 }
 
+export const stripAnsiCodes = (text: string): string => {
+  return text.replace(/\u001b\[.*?m/g, '')
+}
+
```

Comment:
seems oddly simple compared to other ansi stripping snippets I've found online, could we either use a tested package or explain why only that code needs removal?

Reply:
honestly I don't want introduce a new package just for this case, it works just fine for what it's supposed to do